boards/arm/nrf5340dk_nrf5340/nrf5340dk_nrf5340_cpunet.dts

```diff
@@ -186,9 +186,9 @@ arduino_spi: &spi0 {
 			label = "image-scratch";
 			reg = <0x00030000 0xa000>;
 		};
-		storage_partition: partition@3a000 {
+		storage_partition: partition@3f000 {
```

Comment:
This is not only changing the size but also moving the area.
I do not understand, because there is no description of a problem, connection between changing size and moving this area and how it does affect building peripheral_uart. It does not seem that any other area is added here or changed except storage.

Reply:
And the resultant storage partition is actually useless, NACK on this